app/geo/gui/src/ts/twxplore/gui/geo/components/map/Map.tsx

```diff
@@ -78,44 +82,29 @@ const MapImpl: React.FunctionComponent = () => {
             __typename: feature.__typename,
             geometry: feature.geometry,
             frequency: feature.frequency,
-            frequencyUnit: feature.frequencyUnit,
             timestamp: feature.timestamp ? feature.timestamp * 1000 : null,
             postalCode: feature.postalCode,
             transmissionPower: feature.transmissionPower,
             state: MapFeatureState.LOADED,
             frequencyString: feature.frequency
-              ? (Math.floor(feature.frequency * 100) / 100).toString() +
+              ? (
+                  Math.floor((feature.frequency / 1000000) * 100) / 100
+                ).toString() +
                 " " +
-                feature.frequencyUnit
+                "MHz"
               : undefined,
             timestampString: feature.timestamp
               ? new Date(feature.timestamp * 1000).toString()
               : undefined,
             transmissionPowerString: feature.transmissionPower
               ? feature.transmissionPower.toString() + " dB"
               : undefined,
-            Y:
-              feature.type === FeatureType.Transmission
-                ? parseFloat(
-                    feature.geometry.wkt
-                      .replace("POINT", "")
-                      .replace("(", "")
-                      .trim()
-                      .replace(")", "")
-                      .split(" ")[1]
-                  )
-                : undefined,
-            X:
-              feature.type === FeatureType.Transmission
-                ? parseFloat(
-                    feature.geometry.wkt
-                      .replace("POINT", "")
-                      .replace("(", "")
-                      .trim()
-                      .replace(")", "")
-                      .split(" ")[0]
-                  )
-                : undefined,
+            x: (feature.geometry
```

Comment:
Not all features are points so if a feature isn't a point, then x and y will be undefined. Which is fine behavior for me because x and y should only be defined for features that are points. I only use x and y for features that are points.